pkg/k8s/ccnpstatus.go

```diff
@@ -44,14 +44,3 @@ func NewCCNPStatusEventHandler(cnpStore *store.SharedStore, k8sStore cache.Store
 		CNPStatusEventHandler: NewCNPStatusEventHandler(cnpStore, k8sStore, updateInterval),
 	}
 }
-
```

Comment:
On line 42, `func NewCCNPStatusEventHandler` no longer uses `cnpStore` so we can remove it from the function arguments.